ajax/libs/antimoderate/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "antimoderate",
   "version": "0.0.2",
-  "description": "The progressive image loading library for great good!",
-  "filename": "antimoderate.min.js",
+  "description": "The progressive image loading library",
+  "filename": "antimoderate.js",
```

Comment:
Let's use original `antimoderate.min.js`.
We also host `antimoderate.min.js` file, so we have to use `antimoderate.min.js` as filename.
Thanks.